### PR TITLE
fixed wrong paths in verify.ts

### PR DESCRIPTION
**Pull request by philbow61**
changed paths in verify.ts to match repository structure

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3777,6 +3778,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"commander@npm:^8.1.0":
```
Comment (tobikuhlmann) [inline yarn.lock:3781]: is this package used or can it be bumped?

**Review by tobikuhlmann — COMMENTED** [review]
Thank you, that looks good to me. Just one question about the commander package in the comments

File: yarn.lock (hunk 1 above)
Comment (philbow61) [reply yarn.lock:3781]: I guess it can be bumped 👍🏼

Comment (tobikuhlmann) [reply yarn.lock:3781]: there are couple of different versions in there, seems like a dependency, let's keep it in

**Review by tobikuhlmann — APPROVED** [review]

**Pull request merged**